packages/cli/src/commands/ui/create/react.ts

```diff
@@ -5,25 +5,23 @@ import {
 } from '../../../hooks/analytics/analytics';
 import {Config} from '../../../lib/config/config';
 import {platformUrl} from '../../../lib/platform/environment';
-import {
-  spawnProcess,
-  SpawnProcessOutput,
-  spawnProcessOutput,
-} from '../../../lib/utils/process';
-import AuthenticationRequired from '../../../lib/decorators/authenticationRequired';
+import {spawnProcess, spawnProcessOutput} from '../../../lib/utils/process';
 import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
-import {lt as isVersionLessThan} from 'semver';
-import {constants} from 'os';
 import {getPackageVersion} from '../../../lib/utils/misc';
 import {join} from 'path';
-
-const linkToReadme =
-  'https://github.com/coveo/cli/wiki#coveo-uicreatereact-requirements';
+import {
+  Preconditions,
+  IsAuthenticated,
+  IsNodeVersionAbove,
+  IsNpxInstalled,
+} from '../../../lib/decorators/preconditions';
 
 export default class React extends Command {
   static templateName = '@coveo/cra-template';
+  static requiredNodeVersion = '10.16.0';
 
-  static description = `Create a Coveo Headless-powered search page with the React web framework. See ${linkToReadme}`;
+  static description =
```

Comment:
Because the page about the command was really shallow after reworking it, I removed the link.